index.html

```diff
@@ -64,6 +64,23 @@ <h3>
 </p>
 <br />
 
+<h3>
+  What if I want to try GAP right now?
+</h3>
+<p>
+  There are different possibilities for you to test GAP immediately. While none of them will give you access to all features,
+  they all provide valuable functionality. We are aware of
+  <ul>
+    <li><a href="https://tio.run/#gap">TIO</a> (direct cut-and-paste)</li>
+    <li><a href="https://sagecell.sagemath.org/">SageCell</a> (select "gap" in the Language drop-down at the bottom right)</li>
+    <li><a href="https://replit.com/@BlackFingolfin/gap-via-nix#replit.nix">RepLit</a> (a running GAP console)</li>
+    <li>Online computing environments, described <a href="https://www.gap-system.org/Download/alternatives.html">on this page</a>,
+      that already contain GAP: <a href="https://cocalc.com/">CoCalc</a>,
+      <a href="https://community.deepnote.com/c/showcase/using-gap-in-deepnote">DeepNote</a>, ...</li>
```

Comment:
Cocalc and Binder are also mentioned on https://www.gap-system.org/Download/alternatives.html.

I am somewhat reluctant to place these links on the front page without any system in place to make sure they keep working... At the very least perhaps some people could try them to test how well they work right now?

As it is, it seems nobody really is interested in even reviewing GapWWW PRs :-(

Reply:
Concerning the invitation to try the links (perhaps in order to get an idea what one really wants):
- https://tio.run/#gap looks like a pocket calculator for single GAP commands; entering <ctrl>-<enter> twice leads to a GAP error
- https://sagecell.sagemath.org/ seems to work; it is not clear how to deal with the situation that a computation does not finish; for example, `AtlasGroup( "M11" )` hangs (which is not really surprising, according to the comment about "code to freely access internet") 
- https://replit.com/@BlackFingolfin/gap-via-nix#replit.nix looks familiar to a GAP user, and works; for someone who is new to GAP, a short description would be useful
- https://cocalc.com looks confusing: What should one do in order to run an instance of GAP?
- The text at https://community.deepnote.com/c/showcase/using-gap-in-deepnote is perhaps something that should belong to the page about "alternatives" rather than being linked from this page.

Reply:
I agree with Max and Thomas. I found the sagecell to be ok, and replit to be easy to use. tio was confusing. I never found a console or cell or calculator in cocalc or deepnote -- I think cocalc is supposed to have one, but I wasn't sure what deepnote was.
* For replit, can you instead link to a readme that tells the user to press the "Play" or "Run" button? I was briefly confused if I was supposed to edit the file that initially appears. The readme could also link to gap-system.org / getting started stuff in case someone gets to the repl not from the main gap page